packages/next/src/withTRPC.tsx

```diff
@@ -232,6 +239,10 @@ export function withTRPC<
         pageProps.trpcState = trpcClient.runtime.transformer.serialize(
           dehydratedCacheWithErrors,
         );
+        console.log(
+          '---------cache size',
+          dehydratedCacheWithErrors.queries.length,
```

Comment:
For some reason when it errors, it's called twice here but doesn't have any stuff in the cache on the second go. I don't getttt it